packages/react-core/src/core/hooks/wallet-hooks.ts

```diff
@@ -114,3 +122,16 @@ export function useSwitchChain() {
   );
   return context.switchChain;
 }
+
+/**
+ *
+ * @returns a method to set a connected wallet instance
+ */
+export function useSetWallet() {
```

Comment:
Can we rename to `useSetConnectedWallet` ?

Reply:
renamed